project/Dependencies.scala

```diff
@@ -102,6 +106,20 @@ object Dependencies {
     "org.julienrf" %% "endpoints-akka-http-server"    % Versions.endpoints
   )
 
+  private val tapirShared = Seq(
```

Comment:
can you remove tapir part from this PR? I don't want to mix things when it is not necessary

Reply:
Yup, removed